lib/chef/application.rb

```diff
@@ -102,6 +102,12 @@ def load_config_file
       Chef::Config.merge!(config)
     end
 
+    def set_specific_recipes
+      Chef::Config[:specific_recipes] =
```

Comment:
@mcquin it was previously `#=` @ https://github.com/opscode/chef/pull/2710/files#diff-d2d6bb7266330a4a3813a6e01996895dL255